### ISSUE TITLE
AzSqlDatabaseExport Fail into Blob Storage Already Exist

### ISSUE THREAD

**Issue by raphaeldeletoille** [Azure/azure-powershell#21267]
### Description

I've tried to export db bacpac on multiple storage/containers, even with a new storage account from scratch and i always have this return :

New-AzSqlDatabaseExport: The ImportExport operation failed because the storage blob already exists.

It is working fine from Azure Portal but i could not find any reason why i have this error message while the blob container is fine and have no bacpac file in it. Also Network allows Azure Services.

### Issue script & Debug output

```PowerShell
New-AzSqlDatabaseExport -ResourceGroupName *** -ServerName *** -DatabaseName *** -StorageKey *** -StorageUri *** -StorageKeyType StorageAccessKey -AdministratorLogin *** -AdministratorLoginPassword *** -AuthenticationType "SQL"
```

### Environment data

```PowerShell
Name                           Value
----                           -----
PSVersion                      7.2.9
PSEdition                      Core
GitCommitId                    7.2.9
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Module versions

```PowerShell
Script     4.4.0                 Az.Sql                              {Add-AzSqlDatabaseToFailoverGroup, Add-AzSqlElasticJobStep, Add-AzSqlElasticJobTarget, Add-AzSqlInstanceKeyVaultKey…}
```

### Error output

```PowerShell
New-AzSqlDatabaseExport: The ImportExport operation failed because the storage blob already exists.
```

**Comment by SaurabhSharma-MSFT** [comment]
@raphaeldeletoille This looks like an expected behavior.  You need to provide a different backup name if you are exporting under the same container.  You need to provide a different name from Azure portal as well to make it work.  Please let me know if you have any questions.

**Comment by SaurabhSharma-MSFT** [comment]
@raphaeldeletoille We will now proceed to close this thread. If there are further questions regarding this matter, please reopen it and we will gladly continue the discussion.

**Comment by tmfahim** [comment]
Hi there, I am having the same issue. It is failing with blob exist error although I am creating the backup name dynamically.


$fileName = "SQL_DB_BKP_MCS_PRD_$(Get-Date -Format "yyyy-MM-dd-HH-MM").bacpac"
$backpacURL=$storageURL + "/" + $filename

Argument for  New-AzSqlDatabaseExport:
-StorageUri $backpacURL

**Comment by ash3rr** [comment]
@SaurabhSharma-MSFT what do you mean by different backup name? Are you referring to the file referred to in the storageUri?

**Comment by seanvelir** [comment]
I was getting this error when I was using an incorrect storage account name.

**Comment by ash3rr** [comment]
@seanvelir I've copied the URL directly from the properties of the container in the format: https://<account>.blob.core.windows.net/<container>/<file>.bacpac and it still gives the error.

**Comment by OskarKlintrot** [comment]
@ash3rr Did you solve this? I run into the same issue. I try to export to an empty container that I just created but I get ` the storage blob already exists.` anyway.

**Comment by OskarKlintrot** [comment]
I found my issue, I specified `SharedAccessKey` instead of `StorageAccessKey` (I was using the storage access key). @SaurabhSharma-MSFT if this error message isn't the expected maybe consider reopen this issue?

**Comment by ash3rr** [comment]
@OskarKlintrot No, I did not resolve it, and ended up using another solution, but I am now curious to retest it using the SharedAccessKey, as I too was using the StorageAccessKey.

**Comment by OskarKlintrot** [comment]
Sorry, I should have clarified; I used `-StorageKeyType SharedAccessKey` but (in my case) it should be `-StorageKeyType StorageAccessKey` since I used a storage access key from the beginning.

**Comment by soham7kulkarni** [comment]
@ash3rr what did you do to resolve the issue? I tried both types of keys but error persists

**Comment by MichelZ** [comment]
Same issue here. But I am trying to use a SharedAccessKey.
